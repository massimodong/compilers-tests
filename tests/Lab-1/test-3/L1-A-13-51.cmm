float 'hF1Lcom4TM8WAOW [ 947155269 ] , o 

;
int X8SxfI ( ) {
  T1lDL37bokbAkka ;
  Vgn ;
  ;
  K91K6mvmN7jD ;
  if ( SDay90s_tCWnvIT ) {
    wSWqJcdWNTl ;
    if ( lxs )     kim61nMm ;
else     if ( ZF81UI4q070h1nC2xB )     UHQPeB8JLwcBNMGM 
      ;


  }
  if ( RR )   wVbn5qN ;
else   while ( IAS_JBV1 )   return cOQwDL1_HZjnEk 
    


}
!= int aQa ( struct ZBg9 {
  struct {
    int SjvZVOHgPF , c10hbc11ku ';
  }G1Lhtj = rDYK8egiiUdc || Ch6 && WI1kCBp >= z067qbr 
    + SPLfC1xTw ( M67IkT1G = U7UukWbCxQIKsQ2 == TZiBg 
    [ p <= nEbYWoetSMiuk92 - ! ! ( zBMSn9dgO8LnKvX || 
    sd25s > ( fbCD516useamd - pFqnDHZ7J5MR [ p = snAO0 
    || != iJZhYdYDNgMtb1ShIZ9 [ XrRQKtlT5qyhTh2Nh = FA6b5sREzUpde6llh 
    + tS4XVAI9VHtInS8I / ( UT / ( lqGkaQ9GuZO = bVbucw3iQn 
    == s ( g33x4KGif3JAVewN6lO , g1SB ) / - ! TfUE_NWE 
    ) && tdq71Zp ) ] ] ) < 256977762 ) * 959633295.747044266 
    ] * gKjgNfSEl8w ( ) . Zos4NARCVt7 ) / - ! bmicYJnwOF 
    ;
}WqiwxGjQHUbcCq , struct kw8Nqj2PGOAz1_v4 AA1imRyV 
  ) {
}
int ;
