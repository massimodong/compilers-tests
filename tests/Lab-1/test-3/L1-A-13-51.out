Error Type A at Line 1
Error Type B at Line 7
Error Type B at Line 22
Error Type B at Line 23
Error Type A at Line 25
Error Type B at Line 30
